packages/grid/src/vaadin-grid-scroll-mixin.js

```diff
@@ -515,7 +515,7 @@ export const ScrollMixin = (superClass) =>
       });
 
       this.__horizontalScrollPositionStyleSheet.replaceSync(`
-        :host([navigating]) [part~='row']:focus::before {
+        [part~='row']::after {
```

Comment:
Isn't this supposed to be limited to `navigating` and row focus mode?

Reply:
I’d leave this like it is, unless we see it is actually a performance problem. Maybe @tomivirkki can test again?

Reply:
Tested locally and didn't see any performance hit from the selector change. The earlier performance problem was caused by updating a CSS property (even an unused one) on scroll.